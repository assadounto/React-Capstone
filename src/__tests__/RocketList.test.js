import React from 'react';
import '@testing-library/jest-dom';
import TestRenderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import Rockets from '../components/RocketList';

it('Rocket render correctly', () => {
  const Tree = TestRenderer.create(
    <Provider store={store}>
      <Rockets />
    </Provider>,
  );
  expect(Tree).toMatchSnapshot();
});
