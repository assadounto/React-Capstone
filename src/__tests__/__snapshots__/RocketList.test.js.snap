// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rocket render correctly 1`] = `
<div
  className="cont1"
>
  <div
    className="rockets"
  />
</div>
`;
